### PR TITLE
feat: add notes.tpl & ingress to tier

### PR DESCRIPTION
**Pull request by gordalina**
## Summary

This PR fixes the [failed deployment](https://github.com/pluralsh/plural-artifacts/actions/runs/4559430914/jobs/8043383803) because it doesn't have notes.tpl. It also adds ingress to tier.

## Checklist

- [ ]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [ ]  all databases should have dashboards
    - [ ]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [ ]  All major clouds must be supported
    - [ ]  Azure
    - [ ]  AWS
    - [ ]  GCP

### CODE REVIEW

File: tier/plural/recipes/tier-azure.yaml [hunk 1]
```diff
@@ -4,10 +4,15 @@ provider: AZURE
 dependencies:
   - repo: bootstrap
     name: azure-k8s
+  - repo: ingress-nginx
+    name: ingress-nginx-azure
 
 sections:
   - name: tier
-    configuration: []
+    configuration:
```
Comment (michaeljguarino) [inline tier/plural/recipes/tier-azure.yaml:12]: so you'd probably need to add the helm values configuration for this in `tier/helm/tier/values.yaml.tpl` to wire up the ingress appropriately.  I think `strapi` is a pretty good model of this which just uses the helm starter templates to configure a simple stateless app w/ an ingress on top

**Comment by michaeljguarino** [comment]
Also how does tier persist its data?  Does it need a database, I'd assume it has to be saved somewhere.

**Comment by gordalina** [comment]
> Also how does tier persist its data? Does it need a database, I'd assume it has to be saved somewhere.

All data is stored stripe.

File: tier/plural/recipes/tier-azure.yaml (hunk 1 above)
Comment (gordalina) [reply tier/plural/recipes/tier-azure.yaml:12]: Pushed a commit to address this.

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**